composer.json

```diff
@@ -13,12 +13,12 @@
         "php": "^5.6 || ^7.0",
         "doctrine/common": "^2.6",
         "ramsey/uuid": "^3.1",
-        "symfony/event-dispatcher": "^2.6 || ^3.4 || ^4.0",
+        "symfony/event-dispatcher": "^2.6 || ^3.4 || ^4.0 || ^5.0",
         "mtdowling/cron-expression": "^1.1"
```

Comment:
We shoudl update this to the mantained fork and allow the same versions as here: https://github.com/sulu/sulu/blob/dbe800ef7f49936b7846668645040c23281f7b8d/composer.json#L44